src/css/typography.scss

```diff
@@ -223,4 +223,12 @@
   html {
     font-size: 87.5%;
   }
-}
\ No newline at end of file
+}
+
+@media screen and (max-width: 375px) {
+  .homepage {
+    h4 {
+      font-size: var(--h5-font-size);
+    }
+  }
```

Comment:
I mean you should make it generic
`html {
     font-size: 75%`
}`

try to play with it to find better size. If body size looks to small with such configs, then make changes only to headings

Reply:
Yep, with `html { font-size: 75% }` the body font-size is too small and doesn't match the mobile design on figma. Also h4 headings are replaced with h5 headings on mobile design as well, that's why I made changes only to h4 headings